terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1369,6 +1369,10 @@ def qq(*words):
                                 'After=docker.service',
                                 'Requires=docker.service',
                                 '[Service]',
+                                # We set the "Standard…" directives to null to prevent the service from creating
+                                # log messages that are already being handled by docker's logging driver system.
```

Comment:
```suggestion
                                # We explicitly configure Docker (see /etc/docker/daemon.json) to log to journald so we don't need systemd to capture process output
```

And please find evidence/documentation that Docker logging captures both standard out and err.

Reply:
> And please find evidence/documentation that Docker logging captures both standard out and err.

Confirmed docker container stdout & stderr end up in the host's /var/log/messages:

```
[ec2-user@ip-172-71-0-215 ~]$ sudo docker exec -it gitlab /bin/bash
root@gitlab:/#
root@gitlab:/# echo "TEST message to stdout" >> /proc/1/fd/1
root@gitlab:/# echo "TEST message to stderr" >> /proc/1/fd/2
root@gitlab:/# exit
exit
[ec2-user@ip-172-71-0-215 ~]$
[ec2-user@ip-172-71-0-215 ~]$ sudo grep TEST /var/log/messages
Sep 18 04:53:33 ip-172-71-0-215 docker: gitlab: TEST message to stdout
Sep 18 04:53:53 ip-172-71-0-215 docker: gitlab: TEST message to stderr
```